swagger.yaml

```diff
@@ -4966,6 +4966,7 @@ definitions:
       - display_name
       - release_image
       - rhcos_image
+      - rhcos_image_checksum
```

Comment:
Just wonder whether this should be required, or we should skip the validation in case of it isn't specified.

Reply:
@avishayt  can you take a look?

Reply:
Is there any reason not to require it?